### PR TITLE
fix: Upgrade version of jwt-go package to v4.0.0

### PR DESCRIPTION
**Pull request by ayang64**
Address CVE-2020-26160 for the jwt-go package.

Closes #

Describe your proposed changes here.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Feature flagged (if modified API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Review by docmerlin — DISMISSED** [review]
LGTM assuming it passes CI.

File: services/httpd/handler_test.go [hunk 1]
```diff
@@ -196,7 +196,7 @@ func TestHandler_Query_Auth(t *testing.T) {
 	h.ServeHTTP(w, req)
 	if w.Code != http.StatusUnauthorized {
 		t.Fatalf("unexpected status: %d: %s", w.Code, w.Body.String())
-	} else if body := strings.TrimSpace(w.Body.String()); body != `{"error":"signature is invalid"}` {
+	} else if body := strings.TrimSpace(w.Body.String()); body != `{"error":"token signature is invalid"}` {
```
Comment (docmerlin) [inline services/httpd/handler_test.go:199]: Probably shouldn't change the error message, if you don't know if something like Chronograf is relying on it

File: services/httpd/handler_test.go [hunk 2]
```diff
@@ -220,7 +220,7 @@ func TestHandler_Query_Auth(t *testing.T) {
 	h.ServeHTTP(w, req)
 	if w.Code != http.StatusUnauthorized {
 		t.Fatalf("unexpected status: %d: %s", w.Code, w.Body.String())
-	} else if !strings.Contains(w.Body.String(), `{"error":"Token is expired`) {
+	} else if !strings.Contains(w.Body.String(), `{"error":"token is expired`) {
```
Comment (docmerlin) [inline services/httpd/handler_test.go:223]: Probably shouldn't change the error message, if you don't know if something like Chronograf is relying on it

**Review by docmerlin — CHANGES_REQUESTED** [review]

**Review by docmerlin — APPROVED** [review]

File: services/httpd/handler_test.go (hunk 1 above)
Comment (ayang64) [reply services/httpd/handler_test.go:199]: i'm not changing the error message -- the go-jwt package changed *it's* error message.  this change is in a test to reflect this change.  the same error is occurring but the verbiage of the error output has changed.

File: services/httpd/handler_test.go (hunk 2 above)
Comment (ayang64) [reply services/httpd/handler_test.go:223]: i'm not changing the error message -- the go-jwt package changed *it's* error message.  this change is in a test to reflect this change.  the same error is occurring but the verbiage of the error output has changed.

**Pull request merged**